plugins/woocommerce/client/admin/client/wp-admin-scripts/fulfillments/components/shipment-form/test/shipment-tracking-number-form.test.js

```diff
@@ -126,4 +127,55 @@ describe( 'ShipmentTrackingNumberForm', () => {
 			screen.getByPlaceholderText( 'Enter tracking number' )
 		).toBeInTheDocument();
 	} );
+
+	it( 'calls handleTrackingNumberLookup when Enter key is pressed', async () => {
+		mockContext.trackingNumber = '';
+		mockContext.shipmentProvider = '';
+		apiFetch.mockResolvedValueOnce( {
+			tracking_number_details: {
+				tracking_number: '1Z12345E0291980793',
+				shipping_provider: 'ups',
+				tracking_url:
+					'https://www.ups.com/track?tracknum=1Z12345E0291980793',
+			},
+		} );
+		render( <ShipmentTrackingNumberForm /> );
+		const input = screen.getByPlaceholderText( 'Enter tracking number' );
+		fireEvent.change( input, { target: { value: '1Z12345E0291980793' } } );
+		fireEvent.keyDown( input, { key: 'Enter' } );
+
+		await waitFor( () => {
+			expect( mockContext.setTrackingNumber ).toHaveBeenCalledWith(
+				'1Z12345E0291980793'
+			);
+		} );
+		await expect( mockContext.setShipmentProvider ).toHaveBeenCalledWith(
+			'ups'
+		);
+		await expect( mockContext.setTrackingUrl ).toHaveBeenCalledWith(
+			'https://www.ups.com/track?tracknum=1Z12345E0291980793'
+		);
+	} );
+
+	it( 'does not call handleTrackingNumberLookup when Enter key is pressed with empty input', () => {
+		mockContext.trackingNumber = '';
+		render( <ShipmentTrackingNumberForm /> );
+		const input = screen.getByPlaceholderText( 'Enter tracking number' );
+		fireEvent.keyDown( input, { key: 'Enter' } );
+
+		expect( mockContext.setTrackingNumber ).not.toHaveBeenCalled();
+	} );
+
+	it( 'switches to edit mode when tracking number is clicked', () => {
+		mockContext.trackingNumber = '1Z12345E0291980793';
+		render( <ShipmentTrackingNumberForm /> );
+		const trackingNumberSpan = screen.getByRole( 'button', {
+			name: '1Z12345E0291980793',
+		} );
+		fireEvent.click( trackingNumberSpan );
+
+		expect(
+			screen.getByPlaceholderText( 'Enter tracking number' )
+		).toBeInTheDocument();
+	} );
 } );
```

Comment:
Will it also be possible to add test for the chevron opening issue? Since it was an issue till recently.